extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,173 @@
+# KHR_texture_basisu
+
+## Contributors
+
+* Members of 3D Formats Working Group
```

Comment:
I don't see this used anywhere else. Is this what we want to do in the future?

Reply:
Ah, I wrote that as a placeholder initially. @pjcozzi, how should we attribute this extension given its long history?

Reply:
Would be great to explicitly name those that have contributed significantly and then remove this catchall.  When in doubt, let's be generous with credit.